public/src/interfaces/Crystal.ts

```diff
@@ -0,0 +1,6 @@
+export interface Crystal {
+  category: string;
+  photo_URL: string;
```

Comment:
Follow naming convention in the project, use `camelCase` for field and variable names.
```suggestion
  photoUrl: string;
```